change/@ni-nimble-tokens-7d1352e1-f82c-48b8-94fd-d86131297a6d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Comment to prevent accidental merge (think currently confirming in the Blazor Web Chapter that the package will be removed)

Reply:
We've given clients sufficient notice and we can always revive the package in the unlikely event that someone needs it, so I'm going to complete the PR.